src/restapi/fDisconnect.ts

```diff
@@ -27,7 +29,7 @@ export async function fDisconnect(req: Request, res: Response): Promise<void> {
 
     // delete the token from the store
     const tokenStore = newTokenStore(context);
-    const deleteTokenReq = tokenStore.deleteToken(context.acting_user_id);
+    const deleteTokenReq = tokenStore.deleteToken(context.acting_user_id as string);
```

Comment:
The types should make it clear that there is an `actiong_user_id`. Do we need a new type?

Reply:
The token store has since been removed from this PR when Oauth handling was added to the proxy app.  What this question does bring up is the possible need to expose a proxy plugin endpoint to revoke/delete a token